@navikt/core/react/src/util/i18n/locales/nb.ts

```diff
@@ -78,4 +77,32 @@ export default {
     charsTooMany: "{chars} tegn for mye",
     charsLeft: "{chars} tegn igjen",
   },
+  Timeline: {
+    dayFormat: "dd.MM",
+    monthFormat: "MMM yy",
+    yearFormat: "yyyy",
+    Row: {
+      noPeriods: "Ingen perioder",
+      period: "{start} til {end}",
+      dateFormat: "dd.MM.yyyy",
+    },
+    Period: {
+      success: "Suksess",
+      warning: "Advarsel",
+      danger: "Fare",
+      info: "Info",
+      neutral: "Nøytral",
+      period: "{status} fra {start} til {end}",
+      dateFormat: "dd.MM.yyyy",
+    },
+    Pin: {
+      pin: "Pin: {date}",
+      dateFormat: "dd.MM.yyyy",
+    },
+    Zoom: {
+      zoom: "Zoom tidslinjen {start} til {end}",
+      reset: "Tilbakestill zoom",
+      dateFormat: "dd.MM.yyyy",
```

Comment:
Should we have one `dateFormat` under `Timeline` instead of the three nested ones?

Reply:
Yes, think that makes sense 👍